packages/components/src/components/progress-bar-onboarding/progress-bar-onboarding.scss

```diff
@@ -0,0 +1,20 @@
+.progress-bar {
+    display: flex;
+    justify-content: space-evenly;
```

Comment:
shouldn't  this be `center` so can fit in the middle of big containers?
center:
![image](https://user-images.githubusercontent.com/82078941/187596608-88fd102a-4580-447c-bba0-7538882d860d.png)

space-evenly:
![image](https://user-images.githubusercontent.com/82078941/187596486-f8d38660-13b2-4160-aae0-742539cd7f63.png)

Reply:
I have made the changes